Cargo.toml

```diff
@@ -13,4 +13,8 @@ homepage = "https://github.com/Kl4rry/thin-string"
 
 [dependencies]
 thin-vec = "0.2.12"
-serde = { version = "1.0", optional = true }
\ No newline at end of file
+serde = { version = "1.0", optional = true }
```

Comment:
Should serde not be used somewhere?

Reply:
I think it should be ok as-is. It builds successfully with `cargo build --features serde`

Reply:
My bad I got confused by the git diff.